### PR TITLE
Include the sceptre-resolver-cmd

### PR DESCRIPTION
**Pull request by zxiiro**
This includes the sceptre-resolver-cmd
(https://github.com/Sceptre/sceptre-resolver-cmd) into Sceptre as
a core resolver.

Signed-off-by: Thanh Ha <zxiiro@gmail.com>

depends on https://github.com/Sceptre/sceptre-resolver-cmd/pull/5

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ def get_version(rel_path):
     "Jinja2>=2.8,<3",
     "colorama>=0.3.9",
     "packaging>=16.8,<17.0",
+    "sceptre-file-resolver>=1.1.2,<2",
```
Comment (zaro0508) [inline setup.py:31]: this is not the cmd resolver

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (zxiiro) [reply setup.py]: Woops. I updated the version but forgot the name. Fixed.

**Comment by zaro0508** [comment]
there is a [release for sceptre-resolver-cmd](https://pypi.org/project/sceptre-cmd-resolver/) now @zxiiro, would you like to finish this up?

**Review by zaro0508 — APPROVED** [review]

**Comment by zxiiro** [comment]
> there is a [release for sceptre-cmd-resolver](https://pypi.org/project/sceptre-cmd-resolver/) now @zxiiro, would you like to finish this up?

Rebased and updated to 1.1.3.

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**